packages/codemods/nuxt/4/absolute-watch-paths/README.md

```diff
@@ -0,0 +1,29 @@
+# Absolute Watch Paths in `builder:watch`
```

Comment:
i recommend a 1-liner like this at the top:
This codemod converts paths emitted by Nuxt's builder:watch hook from relative to absolute, enhancing support for external directories and complex patterns.